engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/JobPreparationActor.scala

```diff
@@ -60,43 +40,24 @@ case class JobPreparationActor(executionData: WorkflowExecutionActorData,
   }
 
   def prepareJobExecutionActor(inputs: Map[LocallyQualifiedName, WdlValue]): Try[BackendJobPreparationSucceeded] = {
+
+    import RuntimeAttributeDefinition.{addDefaultsToAttributes, evaluateRuntimeAttributes}
+    val curriedAddDefaultsToAttributes = addDefaultsToAttributes(factory.runtimeAttributeDefinitions(initializationData), workflowDescriptor.backendDescriptor.workflowOptions) _
```

Comment:
It means "no really, I want to return this as a function". Otherwise you get a "you didn't supply arguments to this function" error at compile time